experimental/SetPartitions/src/Util.jl

```diff
@@ -0,0 +1,95 @@
+"""
+    helper_new_point_values_vector(p::Vector{Vector{Int}}, q::Vector{Vector{Int}})
+
+Return a semantically identical partition in form of an vector, 
+which has new number values.
+"""
+function helper_new_point_values_vector(p::Vector, q::Vector)
+
+    p_points = vcat(copy(p[1]), copy(p[2]))
```

Comment:
Redundant copy, since you `vcat` anyway?
```suggestion
    p_points = vcat(p[1], p[2])
```